app/webpacker/styles/application.scss

```diff
@@ -20,6 +20,7 @@ $govuk-image-url-function: frontend-image-url;
 @import "related";
 @import "card";
 @import "components/cookie-banner";
+@import "components/start_page_banner";
```

Comment:
Can you rename this to `start-page-banner` to match the hyphenation used by the other components